yogstation/code/controllers/subsystem/input.dm

```diff
@@ -0,0 +1,109 @@
+SUBSYSTEM_DEF(input)
+	name = "Input"
+	wait = 1 //SS_TICKER means this runs every tick
+	init_order = INIT_ORDER_INPUT
+	flags = SS_TICKER
```

Comment:
Is this the ticker that hides the fact that all admins are part of the schutzstaffel. It places a false flag so the players think we're libcucks?